src/server/handlers/github.js

```diff
@@ -150,10 +152,9 @@ export const listRepositories = async (req, res) => {
 
   const body = {
     status: 'success',
-    payload: {
-      code: 'github-list-repositories',
-      repos: response.body
-    }
+    code: 'github-list-repositories',
+    // XXX drop response specific prop, make response a little more general purpose
+    ...normalize(response.body, repoList)
```

Comment:
The 'CALL_API' middleware nests response in `action.payload.response`, so maybe there will be not so many places where we 'manually' push responses in actions (because we will use middleware for most API calls).